### ISSUE TITLE
Creating an email spam classifier.

### ISSUE THREAD

**Issue by vinstex** [section-engineering-education/engineering-education#3352]
### Proposed title of the article
Creating an email spam classifier

### Introduction paragraph (2-3 paragraphs):
There are different kinds of emails, the most common types being spam and ham emails. Ham emails are normal emails, important to you, and do not contain any spam messages. As machine learning engineers, we have to devise a way of distinguishing between the two kinds of emails.

In this article, we will look at how to build a spam classifier using machine learning concepts. This is a continuation of [this](https://www.section.io/engineering-education/email-parser-python/) article by where parsing of emails is looked at.

### Key takeaways: 
In the end, the reader will learn how to use the following tools for the task:
- Linear models(We will use Logistic and Random Forest algorithms for checking the emails).
- NTLK(Natural Language Toolkit ) tool for processing text.
- Cross-validation, precision, and recall for checking the accuracy of the prediction.
> We will use Python.

### References: 
Please list links to any published content/research that you intend to use to support/guide this article.
BOOK: Hands-On Machine Learning with Scikit-Learn & TensorFlow CONCEPTS, TOOLS, AND TECHNIQUES TO BUILD INTELLIGENT SYSTEMS by Aurélien Géron.

**Comment by WanjaMIKE** [comment]
@vinstex 
Good afternoon and thank you for submitting your topic suggestion.
Your topic form has been entered into our queue and should be reviewed (for approval) as soon as a content moderator is finished reviewing the ones in the queue before it.

**Comment by srishilesh** [comment]
@vinstex 

I guess we have plenty of articles on the internet. A vast majority of them uses Linear models and NLTK libraries for spam email classification.

How would yours be different from them? What other additional value do you provide for the reader?
